pkg/app/app.go

```diff
@@ -65,41 +65,40 @@ func ValidateFlags(bundle string, args []string, cmd *cobra.Command, opt *kobjec
 	logrus.Debug("Checking validation of provider %s", provider)
 
 	// OpenShift specific flags
-	deploymentConfig := cmd.Flags().Lookup("deployment-config").Changed
```

Comment:
This code shouldn't be removed. This checks to see if the flag was actually passed in. With your Viper implementation, it does not check _at all_ if the flag was passed.